clients/privacy-center/cypress/e2e/consent.cy.ts

```diff
@@ -403,24 +396,28 @@ describe("Consent settings", () => {
     });
   });
 
-  /**
-   * TODO (PROD-1748): update this test to *override* the config to not rely
-   * on any local ENV settings for the running Privacy Center... right now
-   * this'll fail if you set FIDES_PRIVACY_CENTER__IS_OVERLAY_ENABLED=true on
-   * your local Privacy Center, because the cy.visit("/fides-js-demo.html")
-   * doesn't obey the overrideSettings above.
-   */
   describe("when the user hasn't modified their consent", () => {
     it("reflects the defaults from config.json", () => {
-      cy.visit("/fides-js-demo.html");
+      cy.visit("/fides-js-demo.html?initialize=false");
       cy.get("#consent-json");
-      // eslint-disable-next-line cypress/no-unnecessary-waiting
-      cy.wait(2000);
-      cy.waitUntilFidesInitialized().then(() => {
-        cy.window({ timeout: 500 }).should("have.property", "dataLayer");
-        cy.window().then((win) => {
+      cy.window();
+      cy.window().then((win) => {
```

Comment:
Is the double `.window()` intentional to make this less flakey?

Reply:
I think this might be a copy/pasta issue. I'll double check 😅

Reply:
removed! Good catch.